pkg/controller/kubelet-config/helpers.go

```diff
@@ -227,6 +262,10 @@ func getManagedFeaturesKeyDeprecated(pool *mcfgv1.MachineConfigPool) string {
 	return fmt.Sprintf("98-%s-%s-kubelet", pool.Name, pool.ObjectMeta.UID)
 }
 
+func getManagedNodeKey(pool *mcfgv1.MachineConfigPool, client mcfgclientset.Interface) (string, error) {
+	return ctrlcommon.GetManagedKey(pool, client, "99", "kubelet", getManagedKubeletConfigKeyDeprecated(pool))
```

Comment:
Is this using 99-kubelet-generated-xxx? Wouldn't this conflict with user supplied kubeconfigs?

Reply:
Yeah, updated the code to leverage MachineConfig resource instead of a kubeletConfig resource.